css/mobile-menu.css

```diff
@@ -0,0 +1,50 @@
+.d-flex {
+  display: flex;
+}
+
+.hidden {
+  display: none !important;
+}
+
+.w-100 {
+  width: 100%;
+}
+
+.h-100 {
+  height: 100%;
+}
+
+.dec-none {
+  text-decoration: none;
+}
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  flex-direction: column;
+  padding: 7%;
+  background-color: #3e3c3c;
+  z-index: 11;
+}
+
+.mobile-menu .h-close-btn {
+  cursor: pointer;
+  width: max-content;
+  align-self: flex-end;
+  margin-bottom: 34px;
+}
+
+.mobile-menu .h-link {
+  font-size: 4.7rem;
+  font-style: italic;
+  color: #fff;
+  margin-bottom: 15px;
+}
```

Comment:
As per requirement, please follow the design template from [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Pay attention to the `font-style`, `font-size` and `text-align` rules